components/DrinkNotFound.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { View } from "react-native";
+
+import styled from "styled-components/native";
+
+const DrinkNotFound = () => {
```

Comment:
Nice! I like how you made this its own component for when a drink can't be found.